components/EventItem.tsx

```diff
@@ -1,33 +1,30 @@
 import React from "react"
 import { Image, StyleSheet, Text, View } from "react-native"
 import { Event } from "@lib/events/Event"
-import IconButton from "./common/IconButton"
-import { Divider, Icon } from "react-native-elements"
+import { Divider } from "react-native-elements"
+import { MaterialCommunityIcons } from "@expo/vector-icons"
 import { Shadow } from "react-native-shadow-2"
 import tinycolor from "tinycolor2"
 import { placemarkToFormattedAddress } from "@lib/location"
+import IconButton from "./common/IconButton"
 import { compactFormatMiles } from "@lib/DistanceFormatting"
 
 interface Props {
   event: Event
 }
 
 const EventItem = ({ event }: Props) => {
-  const numAttendees = 1
-  const distance = 0.5
-  const shadowColor = "#bdbdbd"
-  const lightEventColor = tinycolor(event.colorHex).lighten(10).toString()
-
-  const onPressMore = () => {
-    return null
-  }
+  const numAttendees = 1 // Eventually will come from event object
+  const distance = 0.5 // Eventually will be calculated from the location given in the event object
+  const shadowColor = "#ebebeb"
```

Comment:
We could probably use a hue of the event color for the shadow in the future, I think it would look cooler.

Reply:
Yeah, I think that would be pretty cool. I'll experiment with it.